components/neeva/renderer/content_filtering_agent.h

```diff
@@ -85,8 +86,8 @@ class ContentFilteringAgent
   void DeleteOnCorrectThread() const;
 
   scoped_refptr<base::SequencedTaskRunner> task_runner_;
+  
```

Comment:
why these whitespace changes?

Reply:
I thought the service and receiver should be associated with each other